### DISCUSSION TITLE
Xcode 15.0.1 build commit fail

### DISCUSSION THREAD

**Discussion by SGXiaoLin** [codemagic-ci-cd/codemagic-docs#2180] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [x] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

1.Use the platform to pack (default 15.0.1)
2. Submit to connect 

### Expected results

Submit to connect success

### Actual results

Hello, the package of Xcode 15.0.1 version cannot be submitted for review. Can you open the specified 15.0 version?

**Comment by dtrdic** [comment, marked as answer]
Hey @SGXiaoLin, Apple has already [resolved the issue](https://developer.apple.com/forums/thread/740853?answerId=771131022#771131022) related to the submission for review with Xcode version 15.0.1. If you are referring to a different matter, please provide additional details and share the build ID so that I can investigate further
